daphne/src/taskprov.rs

```diff
@@ -0,0 +1,171 @@
+use crate::{
```

Comment:
All files need copyright info. (Feel free to copy-paste from any other Rust source file, they should all be identical at this point.)